frontend/src/Activity/Queue/QueueStatus.js

```diff
@@ -154,8 +154,8 @@ QueueStatus.propTypes = {
 };
 
 QueueStatus.defaultProps = {
-  trackedDownloadStatus: translate('Ok'),
-  trackedDownloadState: translate('Downloading'),
+  trackedDownloadStatus: 'Ok',
```

Comment:
These still should be translated.

This might be a problem for swapping languages while the app is running, but off hand I'm not sure how default props are treated and whether they get evaluated every time they're called on just the first time.

Reply:
OK, i wasn't sure if this one was needed, since its doing a string comparison on lines 51/52 based on what I assume is a query from of the status DL client.
it also seems like trackedDownloadState isn't used here at all?

Reply:
Ahh, they're not strings that should be translated, but values from the API.